README.md

```diff
@@ -214,6 +252,55 @@ It is tempting to design this functionality such that it piggy-backs and/or inte
 
 Therefore, CORS ought to be neither necessary nor sufficient for attaching unpartitioned cookies to a cross-site request. We will therefore design the unpartitioned-cookies-opt-in mechanism as a new thing, completely indepedent from CORS.
 
+### Activation reusability
+
+The following designs were considered as alternatives to the `reuse-for` parameter, to make a storage access activation reusable across requests.
```

Comment:
> The following designs were considered as alternatives to the `reuse-for` parameter,

It looks like some of these are different ways of defining the `reuse-for` parameter, rather than alternatives to the parameter itself. Maybe "alternative designs to allow reuse of activations", to be more general?